web/src/lib/utils/asset-utils.ts

```diff
@@ -59,8 +59,8 @@ export const downloadArchive = async (fileName: string, options: DownloadInfoDto
 
   for (let index = 0; index < downloadInfo.archives.length; index++) {
     const archive = downloadInfo.archives[index];
-    const suffix = downloadInfo.archives.length === 1 ? '' : `+${index + 1}`;
-    const archiveName = fileName.replace('.zip', `${suffix}-${DateTime.now().toFormat('yyyy-LL-dd-HH-mm-ss')}.zip`);
+    const suffix = downloadInfo.archives.length > 1 ? `+${index + 1}` : '';
+    const archiveName = fileName.replace('.zip', `${suffix}-${DateTime.now().toFormat('yyyyLLdd_HHmmss')}.zip`);
```

Comment:
I do prefer the underscore to separate day and hour, but I'm not sure if removing all hyphens makes it better readable.
I would probably vote for
```suggestion
    const archiveName = fileName.replace('.zip', `${suffix}-${DateTime.now().toFormat('yyyy-LL-dd_HH-mm-ss')}.zip`);
```

Reply:
Seems it's matter of taste. I do prefer more compact format, however your suggestion is definetelly better then the original one.